Dockerfile not copied when when building multiple Docker images

**Describe the bug**

When I specify a matrix to build two Docker images (using two different Dockerfiles), `goreleaser` fails with this error:

```
...snip...

   • docker images    
      • building docker image     image=dunglas/mercure:legacy-v0.10.4
   ⨯ release failed after 31.70s error=failed to link dockerfile: link  dist/goreleaserdocker934654131/Dockerfile: no such file or directory
make: *** [build] Error 1
```

**To Reproduce**

`.goreleaser.yml`:

```yaml
before:
  hooks:
  # Create a Caddy skeleton containing the module
  - make prepare
checksum:
  name_template: 'checksums.txt'
snapshot:
  name_template: "{{ .Tag }}-next"
changelog:
  sort: asc
  filters:
    exclude:
    - '^docs:'
    - '^test:'
release:
  prerelease: auto
env:
  - CGO_ENABLED=0
builds:
- id: caddy
  dir: caddy/buildenv
  binary: mercure
  goos:
    - linux
    - darwin
    - windows
  goarch:
    # 32 bits architectures aren't supported by dgraph-io/badger yet:
    # https://github.com/dgraph-io/badger/issues/1385
    #- 386
    - amd64
    #- arm
    - arm64
  goarm:
    - 5
    - 6
    - 7
#Legacy build
- id: legacy
  dir: cmd/mercure
  binary: mercure
  ldflags:
    - -s -w -X github.com/dunglas/mercure/common.version={{ .Version }} -X github.com/dunglas/mercure/common.commit={{ .ShortCommit }} -X github.com/dunglas/mercure/common.buildDate={{ .Date }}
  goos:
    - linux
    - darwin
    - windows
  goarch:
    - 386
    - amd64
    - arm
    - arm64
archives:
- builds:
  - caddy
  replacements:
    darwin: Darwin
    linux: Linux
    windows: Windows
    386: i386
    amd64: x86_64
  files:
  - COPYRIGHT
  - LICENSE
  - README.md
  - public/*
  format_overrides:
  - goos: windows
    format: zip
- id: legacy
  builds:
  - legacy
  name_template: "{{ .ProjectName }}-legacy_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
  replacements:
    darwin: Darwin
    linux: Linux
    windows: Windows
    386: i386
    amd64: x86_64
  files:
  - COPYRIGHT
  - LICENSE
  - README.md
  - public/*
  format_overrides:
  - goos: windows
    format: zip
dockers:
- builds:
  - caddy
  image_templates:
  - 'dunglas/mercure:{{ .Tag }}'
  - 'dunglas/mercure:v{{ .Major }}'
  - 'dunglas/mercure:v{{ .Major }}.{{ .Minor }}'
  - 'dunglas/mercure:latest'
  extra_files:
  - public/
  - Caddyfile
- builds:
  - legacy
  dockerfile: Dockerfile.legacy
  image_templates:
  - 'dunglas/mercure:legacy-{{ .Tag }}'
  - 'dunglas/mercure:legacy-v{{ .Major }}'
  - 'dunglas/mercure:legacy-v{{ .Major }}.{{ .Minor }}'
  - 'dunglas/mercure:legacy-latest'
  extra_files:
  - public/
```

The full code is available here: https://github.com/dunglas/mercure/tree/ci/goreleaser

**Expected behavior**

The Docker image should be built using the custom Dockerfile.

**Environment (please complete the following information):**

- OS: mac
- OS version: Darwin Kernel Version 20.1.0: Sat Oct 31 00:07:11 PDT 2020; root:xnu-7195.50.7~2/RELEASE_X86_64 x86_64
- GoReleaser Version:
goreleaser version 0.147.2
commit: b4f154d81f09ffb73da8c90bbd7f26fba5feac75
built by: homebrew

**Additional context**

```
oreleaser --snapshot --rm-dist --debug

   • debug logs enabled       
   • releasing...     
   • loading config file       file=.goreleaser.yml
   • loaded config file        config={ [CGO_ENABLED=0] {   false false auto  [] []} [] [] { {  } { }      [] } [{caddy [linux darwin windows] [amd64 arm64] [5 6 7] [] [] [] caddy/buildenv  [] [] mercure {[] []} []  [] []  false } {legacy [linux darwin windows] [386 amd64 arm arm64] [] [] [] [] cmd/mercure  [-s -w -X github.com/dunglas/mercure/common.version={{ .Version }} -X github.com/dunglas/mercure/common.commit={{ .ShortCommit }} -X github.com/dunglas/mercure/common.buildDate={{ .Date }}] [] mercure {[] []} []  [] []  false }] [{ [caddy]  map[386:i386 amd64:x86_64 darwin:Darwin linux:Linux windows:Windows]  [{windows zip}]  [COPYRIGHT LICENSE README.md public/*] false} {legacy [legacy] {{ .ProjectName }}-legacy_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }} map[386:i386 amd64:x86_64 darwin:Darwin linux:Linux windows:Windows]  [{windows zip}]  [COPYRIGHT LICENSE README.md public/*] false}] [] [] {{{ .Tag }}-next} {checksums.txt  false} [{[] [caddy]     [dunglas/mercure:{{ .Tag }} dunglas/mercure:v{{ .Major }} dunglas/mercure:v{{ .Major }}.{{ .Minor }} dunglas/mercure:latest]  [public/ Caddyfile] []} {[] [legacy]    Dockerfile.legacy [dunglas/mercure:legacy-{{ .Tag }} dunglas/mercure:legacy-v{{ .Major }} dunglas/mercure:legacy-v{{ .Major }}.{{ .Minor }} dunglas/mercure:legacy-latest]  [public/] []}] [] [] [] [] {{[^docs: ^test:]} asc false}  [] {  } {[make prepare]} {  false} { [] [] [] [] [] []   [] []  {[] []} []  [] []  false } {   false} {  false} { false}}
   • parallelism: 4           
   • loading environment variables
   • getting and validating git state
      • running git               args=[-c log.showSignature=false rev-parse --is-inside-work-tree]
      • git result                stderr= stdout=true

      • running git               args=[-c log.showSignature=false rev-parse --is-inside-work-tree]
      • git result                stderr= stdout=true

      • running git               args=[-c log.showSignature=false show --format='%h' HEAD --quiet]
      • git result                stderr= stdout='a1b80fd'

      • running git               args=[-c log.showSignature=false show --format='%H' HEAD --quiet]
      • git result                stderr= stdout='a1b80fd8578270829f83101d4f524089ab6b7925'

      • running git               args=[-c log.showSignature=false show --format='%ct' HEAD --quiet]
      • git result                stderr= stdout='1606241635'

      • running git               args=[-c log.showSignature=false ls-remote --get-url]
      • git result                stderr= stdout=git@github.com:dunglas/mercure.git

      • running git               args=[-c log.showSignature=false tag --points-at HEAD --sort -version:creatordate]
      • git result                stderr= stdout=
      • running git               args=[-c log.showSignature=false describe --tags --abbrev=0]
      • git result                stderr= stdout=v0.10.4

      • releasing v0.10.4, commit a1b80fd8578270829f83101d4f524089ab6b7925
      • pipe skipped              error=disabled during snapshot mode
   • parsing tag      
   • running before hooks
      • running make prepare
      • rm -Rf dist caddy/buildenv*
cd caddy && XCADDY_SKIP_CLEANUP=1 xcaddy build --with github.com/dunglas/mercure=../../ --with github.com/dunglas/mercure/caddy=../
2020/11/24 19:21:24 [INFO] Temporary folder: /Users/dunglas/workspace/mercure/caddy/buildenv_2020-11-24-1921.945934495
2020/11/24 19:21:24 [INFO] Writing main module: /Users/dunglas/workspace/mercure/caddy/buildenv_2020-11-24-1921.945934495/main.go
2020/11/24 19:21:24 [INFO] Initializing Go module
2020/11/24 19:21:24 [INFO] exec (timeout=10s): /usr/local/bin/go mod init caddy 
go: creating new go.mod: module caddy
2020/11/24 19:21:24 [INFO] Replace github.com/dunglas/mercure => ../../
2020/11/24 19:21:24 [INFO] exec (timeout=10s): /usr/local/bin/go mod edit -replace github.com/dunglas/mercure=../../ 
2020/11/24 19:21:24 [INFO] Replace github.com/dunglas/mercure/caddy => ../
2020/11/24 19:21:24 [INFO] exec (timeout=10s): /usr/local/bin/go mod edit -replace github.com/dunglas/mercure/caddy=../ 
2020/11/24 19:21:24 [INFO] Pinning versions
2020/11/24 19:21:24 [INFO] exec (timeout=0s): /usr/local/bin/go get -d -v github.com/caddyserver/caddy/v2 
go: github.com/caddyserver/caddy/v2 upgrade => v2.2.1
2020/11/24 19:21:25 [INFO] Build environment ready
2020/11/24 19:21:25 [INFO] Building Caddy
2020/11/24 19:21:25 [INFO] exec (timeout=0s): /usr/local/bin/go build -o /Users/dunglas/workspace/mercure/caddy/caddy -ldflags -w -s -trimpath 
go: found github.com/dunglas/mercure in github.com/dunglas/mercure v0.0.0-00010101000000-000000000000
go: found github.com/dunglas/mercure/caddy in github.com/dunglas/mercure/caddy v0.0.0-00010101000000-000000000000
2020/11/24 19:21:26 [INFO] Build complete: caddy
2020/11/24 19:21:26 [INFO] Skipping cleanup as requested; leaving folder intact: /Users/dunglas/workspace/mercure/caddy/buildenv_2020-11-24-1921.945934495
mv caddy/buildenv_* caddy/buildenv
 cmd=make prepare
   • setting defaults 
      • snapshotting     
      • github/gitlab/gitea releases
         • running git               args=[-c log.showSignature=false rev-parse --is-inside-work-tree]
         • git result                stderr= stdout=true

         • running git               args=[-c log.showSignature=false config --get remote.origin.url]
         • git result                stderr= stdout=git@github.com:dunglas/mercure.git

         • pre-release was detected for tag v0.10.4: false
         • pre-release for tag v0.10.4 set to false
      • project name     
      • building binaries
         • skipped invalid build     target=darwin_arm64
         • skipped invalid build     target=windows_arm64
         • skipped invalid build     target=darwin_386
         • skipped invalid build     target=darwin_arm_6
         • skipped invalid build     target=darwin_arm64
         • skipped invalid build     target=windows_arm_6
         • skipped invalid build     target=windows_arm64
      • creating source archive
      • archives         
      • linux packages   
      • snapcraft packages
      • calculating checksums
      • signing artifacts
      • docker images    
      • artifactory      
      • blobs            
      • homebrew tap formula
      • scoop manifests  
      • milestones       
         • running git               args=[-c log.showSignature=false rev-parse --is-inside-work-tree]
         • git result                stderr= stdout=true

         • running git               args=[-c log.showSignature=false config --get remote.origin.url]
         • git result                stderr= stdout=git@github.com:dunglas/mercure.git

   • snapshotting     
   • checking ./dist  
      • ./dist doesn't exist, creating empty folder
   • writing effective config file
      • writing                   config=dist/config.yaml
   • generating changelog
      • pipe skipped              error=not available for snapshots
   • building binaries
      • building                  build={caddy [linux darwin windows] [amd64 arm64] [5 6 7] [] [linux_amd64 linux_arm64 darwin_amd64 windows_amd64] [] caddy/buildenv . [-s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}} -X main.builtBy=goreleaser] [] mercure {[] []} [] go [] []  false go}
      • building                  binary=/Users/dunglas/workspace/mercure/dist/caddy_windows_amd64/mercure.exe
      • building                  binary=/Users/dunglas/workspace/mercure/dist/caddy_linux_arm64/mercure
      • building                  binary=/Users/dunglas/workspace/mercure/dist/caddy_linux_amd64/mercure
      • building                  binary=/Users/dunglas/workspace/mercure/dist/caddy_darwin_amd64/mercure
      • running                   cmd=[go build -ldflags=-s -w -X main.version=v0.10.4-next -X main.commit=a1b80fd8578270829f83101d4f524089ab6b7925 -X main.date=2020-11-24T18:21:23Z -X main.builtBy=goreleaser -o /Users/dunglas/workspace/mercure/dist/caddy_windows_amd64/mercure.exe .] env=[VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 TERM_PROGRAM=vscode LOGNAME=dunglas SHELL=/bin/zsh GOPATH=/Users/dunglas/workspace/go __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less LESS=-R LANG=fr_FR.UTF-8 COLORTERM=truecolor USER=dunglas HOME=/Users/dunglas PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 _=/usr/local/bin/goreleaser TERM=xterm-256color COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners LSCOLORS=Gxfxcxdxbxegedabagacad GOOS=windows GOARCH=amd64 GOARM= GOMIPS= GOMIPS64=]
      • running                   cmd=[go build -ldflags=-s -w -X main.version=v0.10.4-next -X main.commit=a1b80fd8578270829f83101d4f524089ab6b7925 -X main.date=2020-11-24T18:21:23Z -X main.builtBy=goreleaser -o /Users/dunglas/workspace/mercure/dist/caddy_linux_amd64/mercure .] env=[USER=dunglas HOME=/Users/dunglas PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 _=/usr/local/bin/goreleaser COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners LSCOLORS=Gxfxcxdxbxegedabagacad TERM=xterm-256color TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 TERM_PROGRAM=vscode VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 LOGNAME=dunglas SHELL=/bin/zsh GOPATH=/Users/dunglas/workspace/go __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less LESS=-R LANG=fr_FR.UTF-8 COLORTERM=truecolor GOOS=linux GOARCH=amd64 GOARM= GOMIPS= GOMIPS64=]
      • running                   cmd=[go build -ldflags=-s -w -X main.version=v0.10.4-next -X main.commit=a1b80fd8578270829f83101d4f524089ab6b7925 -X main.date=2020-11-24T18:21:23Z -X main.builtBy=goreleaser -o /Users/dunglas/workspace/mercure/dist/caddy_linux_arm64/mercure .] env=[LOGNAME=dunglas SHELL=/bin/zsh GOPATH=/Users/dunglas/workspace/go __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less LESS=-R LANG=fr_FR.UTF-8 COLORTERM=truecolor USER=dunglas HOME=/Users/dunglas PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 _=/usr/local/bin/goreleaser TERM=xterm-256color COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners LSCOLORS=Gxfxcxdxbxegedabagacad VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 TERM_PROGRAM=vscode GOOS=linux GOARCH=arm64 GOARM= GOMIPS= GOMIPS64=]
      • running                   cmd=[go build -ldflags=-s -w -X main.version=v0.10.4-next -X main.commit=a1b80fd8578270829f83101d4f524089ab6b7925 -X main.date=2020-11-24T18:21:23Z -X main.builtBy=goreleaser -o /Users/dunglas/workspace/mercure/dist/caddy_darwin_amd64/mercure .] env=[ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 _=/usr/local/bin/goreleaser COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners LSCOLORS=Gxfxcxdxbxegedabagacad TERM=xterm-256color TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 TERM_PROGRAM=vscode VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 LOGNAME=dunglas SHELL=/bin/zsh GOPATH=/Users/dunglas/workspace/go __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less LESS=-R LANG=fr_FR.UTF-8 COLORTERM=truecolor USER=dunglas HOME=/Users/dunglas PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 GOOS=darwin GOARCH=amd64 GOARM= GOMIPS= GOMIPS64=]
      • added new artifact        name=mercure path=/Users/dunglas/workspace/mercure/dist/caddy_linux_amd64/mercure type=Binary
      • added new artifact        name=mercure.exe path=/Users/dunglas/workspace/mercure/dist/caddy_windows_amd64/mercure.exe type=Binary
      • added new artifact        name=mercure path=/Users/dunglas/workspace/mercure/dist/caddy_linux_arm64/mercure type=Binary
      • added new artifact        name=mercure path=/Users/dunglas/workspace/mercure/dist/caddy_darwin_amd64/mercure type=Binary
      • building                  build={legacy [linux darwin windows] [386 amd64 arm arm64] [6] [] [linux_386 linux_amd64 linux_arm_6 linux_arm64 darwin_amd64 windows_386 windows_amd64] [] cmd/mercure . [-s -w -X github.com/dunglas/mercure/common.version={{ .Version }} -X github.com/dunglas/mercure/common.commit={{ .ShortCommit }} -X github.com/dunglas/mercure/common.buildDate={{ .Date }}] [] mercure {[] []} [] go [] []  false go}
      • building                  binary=/Users/dunglas/workspace/mercure/dist/legacy_windows_amd64/mercure.exe
      • building                  binary=/Users/dunglas/workspace/mercure/dist/legacy_darwin_amd64/mercure
      • building                  binary=/Users/dunglas/workspace/mercure/dist/legacy_windows_386/mercure.exe
      • building                  binary=/Users/dunglas/workspace/mercure/dist/legacy_linux_arm_6/mercure
      • running                   cmd=[go build -ldflags=-s -w -X github.com/dunglas/mercure/common.version=v0.10.4-next -X github.com/dunglas/mercure/common.commit=a1b80fd -X github.com/dunglas/mercure/common.buildDate=2020-11-24T18:21:23Z -o /Users/dunglas/workspace/mercure/dist/legacy_darwin_amd64/mercure .] env=[XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 _=/usr/local/bin/goreleaser TERM=xterm-256color COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners LSCOLORS=Gxfxcxdxbxegedabagacad VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 TERM_PROGRAM=vscode LOGNAME=dunglas SHELL=/bin/zsh GOPATH=/Users/dunglas/workspace/go __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less LESS=-R LANG=fr_FR.UTF-8 COLORTERM=truecolor USER=dunglas HOME=/Users/dunglas PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 GOOS=darwin GOARCH=amd64 GOARM= GOMIPS= GOMIPS64=]
      • running                   cmd=[go build -ldflags=-s -w -X github.com/dunglas/mercure/common.version=v0.10.4-next -X github.com/dunglas/mercure/common.commit=a1b80fd -X github.com/dunglas/mercure/common.buildDate=2020-11-24T18:21:23Z -o /Users/dunglas/workspace/mercure/dist/legacy_windows_386/mercure.exe .] env=[USER=dunglas HOME=/Users/dunglas PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 _=/usr/local/bin/goreleaser COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners LSCOLORS=Gxfxcxdxbxegedabagacad TERM=xterm-256color CGO_ENABLED=0 TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 TERM_PROGRAM=vscode VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js LOGNAME=dunglas SHELL=/bin/zsh GOPATH=/Users/dunglas/workspace/go __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less LESS=-R LANG=fr_FR.UTF-8 COLORTERM=truecolor GOOS=windows GOARCH=386 GOARM= GOMIPS= GOMIPS64=]
      • running                   cmd=[go build -ldflags=-s -w -X github.com/dunglas/mercure/common.version=v0.10.4-next -X github.com/dunglas/mercure/common.commit=a1b80fd -X github.com/dunglas/mercure/common.buildDate=2020-11-24T18:21:23Z -o /Users/dunglas/workspace/mercure/dist/legacy_windows_amd64/mercure.exe .] env=[GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy _=/usr/local/bin/goreleaser __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners LSCOLORS=Gxfxcxdxbxegedabagacad TERM=xterm-256color COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 TERM_PROGRAM=vscode VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 GOPATH=/Users/dunglas/workspace/go LOGNAME=dunglas SHELL=/bin/zsh LESS=-R LANG=fr_FR.UTF-8 COLORTERM=truecolor __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 USER=dunglas HOME=/Users/dunglas GOOS=windows GOARCH=amd64 GOARM= GOMIPS= GOMIPS64=]
      • running                   cmd=[go build -ldflags=-s -w -X github.com/dunglas/mercure/common.version=v0.10.4-next -X github.com/dunglas/mercure/common.commit=a1b80fd -X github.com/dunglas/mercure/common.buildDate=2020-11-24T18:21:23Z -o /Users/dunglas/workspace/mercure/dist/legacy_linux_arm_6/mercure .] env=[XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 _=/usr/local/bin/goreleaser LSCOLORS=Gxfxcxdxbxegedabagacad TERM=xterm-256color COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners TERM_PROGRAM=vscode VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 LOGNAME=dunglas SHELL=/bin/zsh GOPATH=/Users/dunglas/workspace/go COLORTERM=truecolor __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less LESS=-R LANG=fr_FR.UTF-8 USER=dunglas HOME=/Users/dunglas PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 GOOS=linux GOARCH=arm GOARM=6 GOMIPS= GOMIPS64=]
      • added new artifact        name=mercure.exe path=/Users/dunglas/workspace/mercure/dist/legacy_windows_amd64/mercure.exe type=Binary
      • building                  binary=/Users/dunglas/workspace/mercure/dist/legacy_linux_arm64/mercure
      • running                   cmd=[go build -ldflags=-s -w -X github.com/dunglas/mercure/common.version=v0.10.4-next -X github.com/dunglas/mercure/common.commit=a1b80fd -X github.com/dunglas/mercure/common.buildDate=2020-11-24T18:21:23Z -o /Users/dunglas/workspace/mercure/dist/legacy_linux_arm64/mercure .] env=[_=/usr/local/bin/goreleaser __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners LSCOLORS=Gxfxcxdxbxegedabagacad TERM=xterm-256color COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 TERM_PROGRAM=vscode VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 GOPATH=/Users/dunglas/workspace/go LOGNAME=dunglas SHELL=/bin/zsh LESS=-R LANG=fr_FR.UTF-8 COLORTERM=truecolor __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 USER=dunglas HOME=/Users/dunglas GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy GOOS=linux GOARCH=arm64 GOARM= GOMIPS= GOMIPS64=]
      • added new artifact        name=mercure path=/Users/dunglas/workspace/mercure/dist/legacy_darwin_amd64/mercure type=Binary
      • building                  binary=/Users/dunglas/workspace/mercure/dist/legacy_linux_386/mercure
      • running                   cmd=[go build -ldflags=-s -w -X github.com/dunglas/mercure/common.version=v0.10.4-next -X github.com/dunglas/mercure/common.commit=a1b80fd -X github.com/dunglas/mercure/common.buildDate=2020-11-24T18:21:23Z -o /Users/dunglas/workspace/mercure/dist/legacy_linux_386/mercure .] env=[DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners LSCOLORS=Gxfxcxdxbxegedabagacad TERM=xterm-256color COMMAND_MODE=unix2003 XPC_SERVICE_NAME=0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 TERM_PROGRAM=vscode VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ SHELL=/bin/zsh GOPATH=/Users/dunglas/workspace/go LOGNAME=dunglas PAGER=less LESS=-R LANG=fr_FR.UTF-8 COLORTERM=truecolor __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 HOME=/Users/dunglas PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 USER=dunglas SHLVL=1 GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 ORIGINAL_XDG_CURRENT_DESKTOP=undefined OLDPWD=/Users/dunglas/workspace/mercure/caddy KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock XPC_FLAGS=0x0 TERM_PROGRAM_VERSION=1.51.1 _=/usr/local/bin/goreleaser __CFBundleIdentifier=com.microsoft.VSCode GOOS=linux GOARCH=386 GOARM= GOMIPS= GOMIPS64=]
      • added new artifact        name=mercure path=/Users/dunglas/workspace/mercure/dist/legacy_linux_arm_6/mercure type=Binary
      • building                  binary=/Users/dunglas/workspace/mercure/dist/legacy_linux_amd64/mercure
      • running                   cmd=[go build -ldflags=-s -w -X github.com/dunglas/mercure/common.version=v0.10.4-next -X github.com/dunglas/mercure/common.commit=a1b80fd -X github.com/dunglas/mercure/common.buildDate=2020-11-24T18:21:23Z -o /Users/dunglas/workspace/mercure/dist/legacy_linux_amd64/mercure .] env=[TERM_PROGRAM=vscode VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Renderer).app/Contents/MacOS/Code Helper (Renderer) VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js CGO_ENABLED=0 TMPDIR=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/ XPC_SERVICE_NAME=0 ZSH=/Users/dunglas/.oh-my-zsh COMPOSER_MEMORY_LIMIT=-1 LOGNAME=dunglas SHELL=/bin/zsh GOPATH=/Users/dunglas/workspace/go COLORTERM=truecolor __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x1 PAGER=less LESS=-R LANG=fr_FR.UTF-8 USER=dunglas HOME=/Users/dunglas PWD=/Users/dunglas/workspace/mercure GPG_AGENT_INFO=/tmp/gpg-sqChad/S.gpg-agent:494:1 ORIGINAL_XDG_CURRENT_DESKTOP=undefined SHLVL=1 GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh LC_CTYPE=fr_FR.UTF-8 XPC_FLAGS=0x0 OLDPWD=/Users/dunglas/workspace/mercure/caddy KUBECONFIG=/Users/dunglas/.kube/k8s-1-14-1-do-2-ams3-1557312011134-kubeconfig.yaml VSCODE_GIT_IPC_HANDLE=/var/folders/h4/ntrdg2x92wd4348hs9blw0sr0000gn/T/vscode-git-44fd3859f0.sock __CFBundleIdentifier=com.microsoft.VSCode TERM_PROGRAM_VERSION=1.51.1 _=/usr/local/bin/goreleaser LSCOLORS=Gxfxcxdxbxegedabagacad TERM=xterm-256color COMMAND_MODE=unix2003 DISPLAY=/private/tmp/com.apple.launchd.xbfvI5Hfh4/org.macosforge.xquartz:0 PATH=/Users/dunglas/.cargo/bin:/usr/local/opt/libxml2/bin:/usr/local/opt/libiconv/bin:/usr/local/opt/bison/bin:/Users/dunglas/.symfony/bin:/usr/local/opt/sqlite/bin:/usr/local/opt/imagemagick@6/bin:/usr/local/opt/postgresql@9.6/bin:/Users/dunglas/.platformsh/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/dunglas/workspace/go/bin:/Users/dunglas/.composer/vendor/bin:/usr/local/opt/go/libexec/bin:/Users/dunglas/.config/yarn/global/node_modules/.bin/:/Applications/Visual Studio Code.app/Contents/Resources/app/bin SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.dfHpqFbq0E/Listeners GOOS=linux GOARCH=amd64 GOARM= GOMIPS= GOMIPS64=]
      • added new artifact        name=mercure.exe path=/Users/dunglas/workspace/mercure/dist/legacy_windows_386/mercure.exe type=Binary
      • added new artifact        name=mercure path=/Users/dunglas/workspace/mercure/dist/legacy_linux_arm64/mercure type=Binary
      • added new artifact        name=mercure path=/Users/dunglas/workspace/mercure/dist/legacy_linux_amd64/mercure type=Binary
      • added new artifact        name=mercure path=/Users/dunglas/workspace/mercure/dist/legacy_linux_386/mercure type=Binary
   • archives         
      • group linuxarm64 has 1 binaries
      • group darwinamd64 has 1 binaries
      • group linuxamd64 has 1 binaries
      • group windowsamd64 has 1 binaries
      • group linuxamd64 has 1 binaries
      • group linux386 has 1 binaries
      • group windowsamd64 has 1 binaries
      • group darwinamd64 has 1 binaries
      • group linuxarm6 has 1 binaries
      • group windows386 has 1 binaries
      • group linuxarm64 has 1 binaries
      • creating                  archive=dist/mercure_v0.10.4-next_Linux_arm64.tar.gz
      • creating                  archive=dist/mercure_v0.10.4-next_Darwin_x86_64.tar.gz
      • creating                  archive=dist/mercure_v0.10.4-next_Windows_x86_64.zip
      • creating                  archive=dist/mercure_v0.10.4-next_Linux_x86_64.tar.gz
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: LICENSE as LICENSE
      • adding file: LICENSE as LICENSE
      • adding file: LICENSE as LICENSE
      • adding file: LICENSE as LICENSE
      • adding file: README.md as README.md
      • adding file: README.md as README.md
      • adding file: README.md as README.md
      • adding file: README.md as README.md
      • adding file: public/app.css as public/app.css
      • adding file: public/app.css as public/app.css
      • adding file: public/app.css as public/app.css
      • adding file: public/app.css as public/app.css
      • adding file: public/app.js as public/app.js
      • adding file: public/app.js as public/app.js
      • adding file: public/app.js as public/app.js
      • adding file: public/index.html as public/index.html
      • adding file: public/index.html as public/index.html
      • adding file: public/index.html as public/index.html
      • adding file: public/app.js as public/app.js
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: public/index.html as public/index.html
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: /Users/dunglas/workspace/mercure/dist/caddy_linux_amd64/mercure as mercure
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: /Users/dunglas/workspace/mercure/dist/caddy_linux_arm64/mercure as mercure
      • adding file: /Users/dunglas/workspace/mercure/dist/caddy_darwin_amd64/mercure as mercure
      • adding file: /Users/dunglas/workspace/mercure/dist/caddy_windows_amd64/mercure.exe as mercure.exe
      • added new artifact        name=mercure_v0.10.4-next_Linux_arm64.tar.gz path=dist/mercure_v0.10.4-next_Linux_arm64.tar.gz type=Archive
      • creating                  archive=dist/mercure-legacy_v0.10.4-next_Linux_x86_64.tar.gz
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: LICENSE as LICENSE
      • adding file: README.md as README.md
      • adding file: public/app.css as public/app.css
      • adding file: public/app.js as public/app.js
      • adding file: public/index.html as public/index.html
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: /Users/dunglas/workspace/mercure/dist/legacy_linux_amd64/mercure as mercure
      • added new artifact        name=mercure_v0.10.4-next_Windows_x86_64.zip path=dist/mercure_v0.10.4-next_Windows_x86_64.zip type=Archive
      • creating                  archive=dist/mercure-legacy_v0.10.4-next_Windows_x86_64.zip
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: LICENSE as LICENSE
      • adding file: README.md as README.md
      • adding file: public/app.css as public/app.css
      • adding file: public/app.js as public/app.js
      • adding file: public/index.html as public/index.html
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: /Users/dunglas/workspace/mercure/dist/legacy_windows_amd64/mercure.exe as mercure.exe
      • added new artifact        name=mercure_v0.10.4-next_Linux_x86_64.tar.gz path=dist/mercure_v0.10.4-next_Linux_x86_64.tar.gz type=Archive
      • creating                  archive=dist/mercure-legacy_v0.10.4-next_Linux_i386.tar.gz
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: LICENSE as LICENSE
      • adding file: README.md as README.md
      • adding file: public/app.css as public/app.css
      • adding file: public/app.js as public/app.js
      • adding file: public/index.html as public/index.html
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: /Users/dunglas/workspace/mercure/dist/legacy_linux_386/mercure as mercure
      • added new artifact        name=mercure_v0.10.4-next_Darwin_x86_64.tar.gz path=dist/mercure_v0.10.4-next_Darwin_x86_64.tar.gz type=Archive
      • creating                  archive=dist/mercure-legacy_v0.10.4-next_Darwin_x86_64.tar.gz
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: LICENSE as LICENSE
      • adding file: README.md as README.md
      • adding file: public/app.css as public/app.css
      • adding file: public/app.js as public/app.js
      • adding file: public/index.html as public/index.html
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: /Users/dunglas/workspace/mercure/dist/legacy_darwin_amd64/mercure as mercure
      • added new artifact        name=mercure-legacy_v0.10.4-next_Linux_x86_64.tar.gz path=dist/mercure-legacy_v0.10.4-next_Linux_x86_64.tar.gz type=Archive
      • creating                  archive=dist/mercure-legacy_v0.10.4-next_Linux_armv6.tar.gz
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: LICENSE as LICENSE
      • adding file: README.md as README.md
      • adding file: public/app.css as public/app.css
      • adding file: public/app.js as public/app.js
      • adding file: public/index.html as public/index.html
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: /Users/dunglas/workspace/mercure/dist/legacy_linux_arm_6/mercure as mercure
      • added new artifact        name=mercure-legacy_v0.10.4-next_Linux_i386.tar.gz path=dist/mercure-legacy_v0.10.4-next_Linux_i386.tar.gz type=Archive
      • creating                  archive=dist/mercure-legacy_v0.10.4-next_Linux_arm64.tar.gz
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: LICENSE as LICENSE
      • adding file: README.md as README.md
      • adding file: public/app.css as public/app.css
      • adding file: public/app.js as public/app.js
      • adding file: public/index.html as public/index.html
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: /Users/dunglas/workspace/mercure/dist/legacy_linux_arm64/mercure as mercure
      • added new artifact        name=mercure-legacy_v0.10.4-next_Linux_armv6.tar.gz path=dist/mercure-legacy_v0.10.4-next_Linux_armv6.tar.gz type=Archive
      • creating                  archive=dist/mercure-legacy_v0.10.4-next_Windows_i386.zip
      • adding file: COPYRIGHT as COPYRIGHT
      • adding file: LICENSE as LICENSE
      • adding file: README.md as README.md
      • adding file: public/app.css as public/app.css
      • adding file: public/app.js as public/app.js
      • adding file: public/index.html as public/index.html
      • adding file: public/mercure.svg as public/mercure.svg
      • adding file: /Users/dunglas/workspace/mercure/dist/legacy_windows_386/mercure.exe as mercure.exe
      • added new artifact        name=mercure-legacy_v0.10.4-next_Windows_x86_64.zip path=dist/mercure-legacy_v0.10.4-next_Windows_x86_64.zip type=Archive
      • added new artifact        name=mercure-legacy_v0.10.4-next_Darwin_x86_64.tar.gz path=dist/mercure-legacy_v0.10.4-next_Darwin_x86_64.tar.gz type=Archive
      • added new artifact        name=mercure-legacy_v0.10.4-next_Linux_arm64.tar.gz path=dist/mercure-legacy_v0.10.4-next_Linux_arm64.tar.gz type=Archive
      • added new artifact        name=mercure-legacy_v0.10.4-next_Windows_i386.zip path=dist/mercure-legacy_v0.10.4-next_Windows_i386.zip type=Archive
   • creating source archive
      • pipe skipped              error=source pipe is disabled
   • linux packages   
   • snapcraft packages
   • calculating checksums
      • checksumming              file=mercure-legacy_v0.10.4-next_Windows_i386.zip
      • calculating checksum for dist/mercure-legacy_v0.10.4-next_Windows_i386.zip
      • checksumming              file=mercure_v0.10.4-next_Darwin_x86_64.tar.gz
      • calculating checksum for dist/mercure_v0.10.4-next_Darwin_x86_64.tar.gz
      • checksumming              file=mercure_v0.10.4-next_Windows_x86_64.zip
      • calculating checksum for dist/mercure_v0.10.4-next_Windows_x86_64.zip
      • checksumming              file=mercure_v0.10.4-next_Linux_arm64.tar.gz
      • calculating checksum for dist/mercure_v0.10.4-next_Linux_arm64.tar.gz
      • checksumming              file=mercure_v0.10.4-next_Linux_x86_64.tar.gz
      • calculating checksum for dist/mercure_v0.10.4-next_Linux_x86_64.tar.gz
      • checksumming              file=mercure-legacy_v0.10.4-next_Darwin_x86_64.tar.gz
      • calculating checksum for dist/mercure-legacy_v0.10.4-next_Darwin_x86_64.tar.gz
      • checksumming              file=mercure-legacy_v0.10.4-next_Windows_x86_64.zip
      • calculating checksum for dist/mercure-legacy_v0.10.4-next_Windows_x86_64.zip
      • checksumming              file=mercure-legacy_v0.10.4-next_Linux_x86_64.tar.gz
      • calculating checksum for dist/mercure-legacy_v0.10.4-next_Linux_x86_64.tar.gz
      • checksumming              file=mercure-legacy_v0.10.4-next_Linux_i386.tar.gz
      • calculating checksum for dist/mercure-legacy_v0.10.4-next_Linux_i386.tar.gz
      • checksumming              file=mercure-legacy_v0.10.4-next_Linux_arm64.tar.gz
      • calculating checksum for dist/mercure-legacy_v0.10.4-next_Linux_arm64.tar.gz
      • checksumming              file=mercure-legacy_v0.10.4-next_Linux_armv6.tar.gz
      • calculating checksum for dist/mercure-legacy_v0.10.4-next_Linux_armv6.tar.gz
      • added new artifact        name=checksums.txt path=dist/checksums.txt type=Checksum
   • signing artifacts
   • docker images    
      • looking for binaries matching docker={[] [legacy] linux amd64  Dockerfile.legacy [dunglas/mercure:legacy-{{ .Tag }} dunglas/mercure:legacy-v{{ .Major }} dunglas/mercure:legacy-v{{ .Major }}.{{ .Minor }} dunglas/mercure:legacy-latest]  [public/] []}
      • found binaries            binaries=[]
      • looking for binaries matching docker={[] [caddy] linux amd64   [dunglas/mercure:{{ .Tag }} dunglas/mercure:v{{ .Major }} dunglas/mercure:v{{ .Major }}.{{ .Minor }} dunglas/mercure:latest]  [public/ Caddyfile] []}
      • found binaries            binaries=[]
      • tempdir: dist/goreleaserdocker476116507
      • tempdir: dist/goreleaserdocker477260478
      • extra file                dst=dist/goreleaserdocker476116507/public src=public/
      • extra file                dst=dist/goreleaserdocker476116507/public/app.css src=public/app.css
      • extra file                dst=dist/goreleaserdocker476116507/public/app.js src=public/app.js
      • extra file                dst=dist/goreleaserdocker476116507/public/index.html src=public/index.html
      • extra file                dst=dist/goreleaserdocker476116507/public/mercure.svg src=public/mercure.svg
      • building docker image     image=dunglas/mercure:legacy-v0.10.4
      • running                   cmd=[docker build . -t dunglas/mercure:legacy-v0.10.4 -t dunglas/mercure:legacy-v0 -t dunglas/mercure:legacy-v0.10 -t dunglas/mercure:legacy-latest] cwd=dist/goreleaserdocker476116507
   ⨯ release failed after 36.23s error=failed to link dockerfile: link  dist/goreleaserdocker477260478/Dockerfile: no such file or directory
```

--- a/internal/pipe/docker/docker.go
+++ b/internal/pipe/docker/docker.go
@@ -40,6 +40,9 @@ func (Pipe) Default(ctx *context.Context) error {
 		if docker.Goarch == "" {
 			docker.Goarch = "amd64"
 		}
+		if docker.Dockerfile == "" {
+			docker.Dockerfile = "Dockerfile"
+		}
 		for _, f := range docker.Files {
 			if f == "." || strings.HasPrefix(f, ctx.Config.Dist) {
 				return fmt.Errorf("invalid docker.files: can't be . or inside dist folder: %s", f)
@@ -54,9 +57,6 @@ func (Pipe) Default(ctx *context.Context) error {
 		ctx.Config.Dockers[0].Binaries = []string{
 			ctx.Config.Builds[0].Binary,
 		}
-	}
-	if ctx.Config.Dockers[0].Dockerfile == "" {
-		ctx.Config.Dockers[0].Dockerfile = "Dockerfile"
 	}
 	return nil
 }

--- a/internal/pipe/docker/docker_test.go
+++ b/internal/pipe/docker/docker_test.go
@@ -725,6 +725,24 @@ func TestDefault(t *testing.T) {
 	require.Empty(t, docker.Builds)
 }
 
+func TestDefaultDockerfile(t *testing.T) {
+	var ctx = &context.Context{
+		Config: config.Project{
+			Builds: []config.Build{
+				{},
+			},
+			Dockers: []config.Docker{
+				{},
+				{},
+			},
+		},
+	}
+	require.NoError(t, Pipe{}.Default(ctx))
+	require.Len(t, ctx.Config.Dockers, 2)
+	require.Equal(t, "Dockerfile", ctx.Config.Dockers[0].Dockerfile)
+	require.Equal(t, "Dockerfile", ctx.Config.Dockers[1].Dockerfile)
+}
+
 func TestDefaultBinaries(t *testing.T) {
 	var ctx = &context.Context{
 		Config: config.Project{

